setup.py

```diff
@@ -79,5 +80,5 @@
         package_data={"asynq": DATA_FILES},
         ext_modules=EXTENSIONS,
         setup_requires=["Cython", "qcore", "setuptools"],
-        install_requires=["qcore", "pygments"],
+        install_requires=["Cython", "qcore", "pygments"],
```

Comment:
I just removed Cython from the runtime dependencies and I'd rather keep it that way. You can use a `.pxd` file to replace the runtime use of Cython.